nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nuxeo.defaults

```diff
@@ -1,5 +1,6 @@
 ## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+nuxeo.template.includes=s3binaries
 #GLACIER
-s3-coldstorage.target=.
-nuxeo.glacier.bucket=
-nuxeo.glacier.bucket_prefix=
+coldstorage.target=.
+coldstorage.bucket=
+coldstorage.bucket_prefix=
```

Comment:
I think we should keep the naming `nuxeo.glacier` as here we are using them as s3 bucket. Should we even add `aws` => `nuxeo.aws.glacier.bucket`